### PR TITLE
updated readme to mention verilator dependency 

### PR DESCRIPTION
**Pull request by anniej-sifive**
### Contributor Checklist

- [ ] Did you add Scaladoc to every public function/method?
- [ ] Did you add at least one test demonstrating the PR?
- [ ] Did you delete any extraneous printlns/debugging code?
- [x] Did you specify the type of improvement?
- [ ] Did you add appropriate documentation in `docs/src`?
- [x] Did you state the API impact?
- [x] Did you specify the code generation impact?
- [x] Did you request a desired merge strategy?
- [ ] Did you add text to be included in the Release Notes for this change?

#### Type of Improvement

 - documentation 

#### API Impact

None

#### Backend Code Generation Impact

None

#### Desired Merge Strategy

- Squash: The PR will be squashed and merged (choose this if you have no preference. 

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels?
- [ ] Did you mark the proper milestone (3.2.x, 3.3.x, 3.4.x, 3.5.0) ?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?
- [ ] Did you mark as `Please Merge`?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,6 +169,8 @@ git clone https://github.com/chipsalliance/chisel3.git
 cd chisel3
 sbt compile
 ```
+In order to run the following unit tests, make sure you have Verilator installed and loaded
+(Check this by running 'which verilator')
```
Comment (debs-sifive) [inline README.md:173]:
```suggestion

In order to run the following unit tests, make sure you have Verilator installed and loaded
(you can check this by running 'which verilator').
```

Comment (jackkoenig) [reply README.md:173]:
```suggestion
In order to run the following unit tests, make sure you have Verilator installed and on your `PATH`
(Check this by running `which verilator`).
```

Comment (mwachs5) [reply README.md:173]: I like Jack's suggestion better

Comment (debs-sifive) [reply README.md:173]: i think they can both be used together; i'm just suggesting that the capitalization be fixed in the parentheses.

Comment (jackkoenig) [reply README.md:173]: I've incorporated your change into mine.

**Review by jackkoenig — APPROVED** [review]

**Pull request merged**